### DISCUSSION TITLE
[OP-Stack] Fraud Proof Update of the OP-stack

### DISCUSSION THREAD

**Discussion by yao-jz** [ethereum-optimism/developers#292] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hey guys,

I want to open a op-stack chain with fraud proof cannon (or asterisc). For the fraud proof process, I want to use cannon/asterisc to fraud proof any possible program(rather than only for rollup). I have successfully compiled and run my customized program to assembly and in the local VM to generate proof. 

I found that the Fault Proof feature will be online in a few days. So does op-stack for now supports me to use the fraud proof, verify the proof and perform the challenge game on chain for my specific program? If so, which version or commit in this mono repo supports me to do the modification? Thanks!

**Comment by Chomtana** [comment]
It should be supported in the latest 1.7.2 of op-node but the official document hasn't come out yet

You may look into deploy-config generation script for a sneak peek

https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/scripts/getting-started/config.sh

**Reply by yao-jz** [reply]
Thanks! So do you mean I can use 1.7.2 version of op-node, op-batcher and op-proposer together to add support for fraud proof?

Yes, I can find a ChallengePeriod config in that file.

**Reply by Chomtana** [reply]
I think so but it isn't official yet
